graphql.install

```diff
@@ -37,10 +37,12 @@ function graphql_uninstall(): void {
   $configFactory = \Drupal::getContainer()->get('config.factory');
   $languageTypes = $configFactory->getEditable('language.types');
   $negotiation = $languageTypes->get('negotiation');
-  foreach (array_keys($negotiation) as $type) {
-    unset($negotiation[$type]['enabled']['language-graphql']);
+  if (!empty($negotiation)) {
+    foreach (array_keys($negotiation) as $type) {
+      unset($negotiation[$type]['enabled']['language-graphql']);
+    }
+    $languageTypes->set('negotiation', $negotiation)->save();
```

Comment:
I understand the idea but I think the real issue here is that we're trying to modify config from a config entity that may not exist (because the `language` module might not be enabled). So rather than introducing an `empty` check (which I find too broad in general), I think adding a check before we even get to `$negotiation` works better. We're guaranteed by the config schema that `negotiation` is an array.

My suggestion would be to make this something like:
```php
if (!$languageTypes->isNew()) {
  $negotiation = ...
  ...
  $languageTypes->set(..)->save();
}
```

Reply:
Thanks for the review. I tried this out, works like a charm. I have made the changes and pushed in code